src/web3data.js

```diff
@@ -94,6 +94,46 @@ class Web3Data {
     this.websocket.off({eventName, filters}, callback)
   }
 
+  getGasPrice() {
+    return this.block.getGasPrice()
+  }
+
+  getBlockNumber() {
+    return this.block.getBlockNumber()
+  }
+
+  getCode(hash) {
+    return this.contract.getCode(hash)
+  }
+
+  getBlock(id, filterOptions) {
+    return this.block.getBlock(id, filterOptions)
+  }
+
+  getUncle(id, index) {
+    return this.block.getUncle(id, index)
+  }
+
+  getBlockTransactionCount(id) {
+    return this.block.getBlockTransactionCount(id)
+  }
+
+  getTransactionFromBlock(id, index) {
+    return this.block.getTransactionFromBlock(id, index)
+  }
+
+  getTransactions(id, filterOptions) {
+    return this.block.getTransactions(id, filterOptions)
+  }
+
+  async getEtherPrice() {
+    const response = await get(this, {
+      endpoint: '/market/prices/eth/latest'
+    })
+    throwIf(!response || response.status !== 200 || !response.payload)
+    return response.payload
```

Comment:
@taylorjdawson what payload is expected here? Im fine returning all of our prices, just want to double check here since the wiki didn't specify.

Reply:
Also, this method should live under `.eth`